projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/BlackBondFuturesExpLogMoneynessProviderDiscount.java

```diff
@@ -18,24 +18,26 @@
    * @param parameters The Black parameters.
    * @param legalEntity The legal entity of the bonds underlying the futures for which the volatility data is valid.
    */
-  public BlackBondFuturesExpLogMoneynessProviderDiscount(final IssuerProviderDiscount issuerProvider, final Surface<Double, Double, Double> parameters, final LegalEntity legalEntity) {
+  public BlackBondFuturesExpLogMoneynessProviderDiscount(final IssuerProviderInterface issuerProvider,
+      final Surface<Double, Double, Double> parameters, final LegalEntity legalEntity) {
     super(issuerProvider, parameters, legalEntity);
   }
 
   @Override
   public BlackBondFuturesExpLogMoneynessProviderDiscount copy() {
-    final IssuerProviderDiscount issuerProvider = getIssuerProvider().copy();
+    IssuerProviderInterface issuerProvider = getIssuerProvider().copy();
     return new BlackBondFuturesExpLogMoneynessProviderDiscount(issuerProvider, getBlackParameters(), getLegalEntity());
   }
 
   @Override
   public MulticurveProviderDiscount getMulticurveProvider() {
-    return getIssuerProvider().getMulticurveProvider();
+    IssuerProviderInterface issuerProvider = getIssuerProvider();
```

Comment:
Why this "if"? The original code does not work anymore? I should be a direct pass-through of the underlying curves.